Jenkinsfile

```diff
@@ -153,6 +153,11 @@ pipeline {
             }
         }
         stage('EtherCAT and no-connection tests') {
+            when {
+                expression {
```

Comment:
Consider adding a TODO to remove this condition when the machine is ready.

Reply:
Done